source/_elephpants/2017-10-06-purple-heroku.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Hero
+categories:
+    - heroku
+tags:
+    - purple
+sponsor: Heroku
+reverse: Heroku logo
+photos:
+    -
+        file: purple-heroku-0-400x300.jpg
+        link: https://www.flickr.com/photos/atomictaco/38171946464/
+        credit: Rob Allen
```

Comment:
Is this Rob Allen's photo?

Reply:
Copy paste fail; thanks for catching

Reply:
Also, do we need/have permission from "atomictaco" that these images can be used?

Reply:
I have several personas online; I am also atomictaco.
Regardless, the photos are CC-BY-SA and are only linked as per request from [CONTRIBUTING](https://github.com/philipsharp/afieldguidetoelephpants/blob/master/CONTRIBUTING.md).

Reply:
:+1: awesome! :)